lib/vm-util

```diff
@@ -175,7 +175,7 @@ Usage: vm ...
     datastore add <name> <path>
     list [-r]
     info [name] [...]
-    create [-d datastore] [-t template] [-s size] [-m memory] [-c vCPUs] [-i vm-image] [-C -k pubkeys] [-n "netconfig"] <name>
+    create [-d datastore] [-t template] [-s size] [-m memory] [-c vCPUs] [-i vm-image] [-C [-k pubkeys | -u user-data] [-n "netconfig"]] <name>
```

Comment:
should both say `"...-file"`?

Reply:
You've got a point. But let's go ahead and change them later if necessary.